dev/bench/gitlab.sh

```diff
@@ -70,8 +74,9 @@ else
     exit 1
 fi
 
-mkdir -p "_bench"
-working_dir="$PWD/_bench"
+bench_dirname="_bench"
+mkdir -p "${bench_dirname}"
```

Comment:
No need for curlies here
```suggestion
mkdir -p "$bench_dirname"
```
and other occurences

Reply:
Does it hurt though?

Reply:
meh